lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Sign.hs

```diff
@@ -295,6 +328,13 @@ estimateKeyWitnessCount utxo tx =
                     , "Caller is expected to ensure this does not happen."
                     ]
 
+newtype SpecifiedTimelockScriptVkCounts = SpecifiedTimelockScriptVkCounts
+    { getSpecifiedTimelockScriptVkCounts
+        :: (Map (ScriptHash StandardCrypto) Natural)
+    }
+    deriving (Show, Eq)
+    deriving newtype (Monoid, Semigroup)
```

Comment:
I'm curious, what semantics do you intend for this `Semigroup` instance to have?

According to the default behaviour of `Data.Map`, `<>` is left-biased, so the following expressions may evaluate to different values:
- `m1 <> m2`
- `m2 <> m1`

Reply:
I don't think it matters. I added a comment: 8e294d9548